src/components/Verse/index.js

```diff
@@ -16,14 +16,14 @@ import debug from 'helpers/debug';
 
 import { loadTafsirs } from '../../redux/actions/media';
 
-const Copy = Loadable({
-  loader: () => import(/* webpackChunkName: "Copy" */ 'components/Copy'),
-  LoadingComponent: ComponentLoader
+const Copy = asyncComponent({
```

Comment:
just curious what the reason behind this change?

Reply:
`Loadable` vs  `asyncComponent`? It's cause we have 2-3 different libraries for async components and I'd like to get the bundlesize down. `async-component` felt like it was more popular than `loadable-components` and easier to work with on the server than `Loadable`